### PR TITLE
more small pillowtop cleanup

### PR DESCRIPTION
**Pull request by czue**
@snopoke @dannyroberts cc @proteusvacuum 

I believe these are all small and safe changes, though am open to push back on that given past issues. I was pretty careful testing every path of `run_ptop`

can be read commit by commit

### CODE REVIEW

**Comment by dannyroberts** [comment]
(not mergeable)

**Comment by czue** [comment]
thanks, fixed

**Comment by czue** [comment]
:white_check_mark:

File: pillowtop/utils.py [hunk 1]
```diff
@@ -70,14 +71,11 @@ def get_pillow_config_from_setting(section, pillow_config_string_or_dict):
...
-                if pillow_class_name in full_str:
-                    if instantiate:
-                        return get_pillow_instance(full_str)
-                    else:
-                        return get_pillow_class(full_str)
+    all_configs = get_all_pillow_configs()
+    for config in all_configs:
+        if config.name == pillow_class_name:
+            return config.get_instance() if instantiate else config.get_class()
+    raise PillowNotFoundError(u'No pillow found with name {}'.format(pillow_class_name))
```
Comment (benrudolph) [inline pillowtop/utils.py]: this is such an improvement over `ValueError`. that made no sense

**Pull request merged**